vendor/licenses/grammar/Syntax-highlighting-for-PostCSS.txt

```diff
@@ -0,0 +1,5 @@
+---
+type: grammar
+name: Syntax-highlighting-for-PostCSS
+license: other
```

Comment:
Try `license: mit` here. Maybe it'll fix CI fail.

And maybe you should copy license text as well. Other files in this directory have license text in them.